compatibility.md

```diff
@@ -0,0 +1,26 @@
+# Platform Compatibility for OSBAPI
+
+
+| Release or Feature | Introduced | Cloud Foundry | Kubernetes |
+| --- | -- | --- | --- |
+| Plan Bindable | v2.11 | ✔️ | ✔️ |
+| [*v2.11*](release-notes.md#v211) | Aug 11, 2016 | ✔️ | ✔️ |
+| `context` for PUT,PATCH instance | v2.12 | ✔️ | ✔️ |
+| [*v2.12*](release-notes.md#v212) | June 13, 2017 | ✔️ | ✔️ |
+| `schemas` in catalog | v2.13 | ✔️ | ✔️ |
+| `context` for PUT binding | v2.13 | ✔️ | ✔️ |
+| `volume_mounts` bindings | v2.13 | ✔️ | - |
+| `originating identity` header | v2.13 | ✔️ | ✔️ |
+| [*v2.13*](release-notes.md#v213) | Sep 27, 2017 | ✔️ | ✔️ |
+| `clusterid` in context | v2.14 | - | ✔️ |
```

Comment:
This is a Kube-specific field so I wonder if its really fair to show it in the table since people might ask when (or "if") CF will support it.

Reply:
the "n/a" idea you mentioned is a good one

Reply:
perhaps say something like:  `clusterid` in context for Kubernetes

and then put NA for CF

Reply:
would `context.clusterid` be easier/faster for people to parse?

Reply:
Fair, I will update.